OpenKeychain/src/main/java/org/sufficientlysecure/keychain/keyimport/ImportKeysListEntry.java

```diff
@@ -82,6 +88,10 @@ public String getKeyIdHex() {
         return mKeyIdHex;
     }
 
+    public long getKeyId() {
+        return new BigInteger(mKeyIdHex.substring(2), 16).longValue();
```

Comment:
Better put this in one of the util classes, I think https://github.com/open-keychain/open-keychain/blob/master/OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/util/KeyFormattingUtils.java already contains similar methods.